src/components/bot-notification/bot-notification-utils.ts

```diff
@@ -44,6 +44,7 @@ export const notification_message = () => ({
     [NOTIFICATION_TYPE.BOT_DELETE]: localize('You’ve successfully deleted a bot.'),
     strategy_conversion: localize('Save this strategy as an XML file from Deriv Bot for faster re-imports.'),
     google_drive_error: localize('Your session has expired. Please sign in again.'),
+    success_quick_strategy_message: localize(`You've successfully imported a bot.`),
```

Comment:
Why are we creating another notification message? We already have NOTIFICATION_TYPE.BOT_IMPORT which has the same message. Lets use that @mayuran-deriv